cogs/economy.py

```
@@ -0,0 +1,319 @@
+import discord
+from discord.ext import commands
+import random
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import

(F401)